configs/platforms/amazon-2023-aarch64.rb

```diff
@@ -0,0 +1,20 @@
+# rubocop:disable Naming/FileName
+# frozen_string_literal: true
```

Comment:
I don't think that you need these Rubocop lines, we don't require Rubocop in this repo:

https://github.com/puppetlabs/puppet-runtime/blob/50ebe6520e0ee418455be5a6fc3ca6dc22163531/Gemfile#L25

Reply:
Raised maintenance PR (https://github.com/puppetlabs/puppet-runtime/pull/749) to address this